actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -404,6 +407,9 @@ def cookies
     end
 
   private
+    def valid_http_status?(status)
+      !status.is_a?(Symbol) || Rack::Utils::SYMBOL_TO_STATUS_CODE.include?(status)
```

Comment:
I don't think we need to check if it is a symbol or not. If it is not a symbol ` Rack::Utils::SYMBOL_TO_STATUS_CODE.include?(status)` will return false.